.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,53 @@
+name: Documentation
+description: File a documentation request/report
+title: "[Docs:]: "
```

Maybe I misspelled these
```suggestion
title: "[Feat]: "
```